packages/docgen/lib/get-jsdoc-from-token.js

```diff
@@ -25,10 +25,15 @@ module.exports = ( token ) => {
 			spacing: 'preserve',
 		} )[ 0 ];
 		if ( jsdoc ) {
+			let paramCount = 0;
 			jsdoc.tags = jsdoc.tags.map( ( tag ) => {
+				const isUnqualifiedParam =
+					tag.tag === 'param' && ! tag.name.includes( '.' );
```

Comment:
What does `tag.tag === 'param'` mean? Is that just indicating it's the array parameter?

Reply:
It just indicates that it is a param tag (`@param {type} name Description.`)